confmap/README.md

```diff
@@ -102,3 +102,60 @@ configuration retrieved via the `Provider` used to retrieve the “initial” co
 ```
 
 The `Resolver` does that by passing an `onChange` func to each `Provider.Retrieve` call and capturing all watch events. 
+
+## Troubleshooting
```

Comment:
I wonder if this should go in the more general troubleshooting guide

Reply:
Added a link from the main troubleshooting doc

Reply:
Thanks!